m01_introduction/index.rst

```diff
@@ -5,12 +5,11 @@ Introduction to developing course on A-Plus learning management service.
 
 .. toctree::
 
-  01_installation
-  0X_gallery
-  02_rst
-  03_css
-  04_git
-  05_docker
-  06_languages
-  07_questionnaires
+  01_gallery
+  02_languages
```

Comment:
I thought that it is strange that the chapters about multilingual courses and questionnaires are now placed before any introduction to RST and the `toctree` directive. Why did you decide that readers should study multilingual courses before any other topic? Most courses are not multilingual.